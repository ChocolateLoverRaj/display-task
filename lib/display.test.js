/* global test, expect */

// There are no actual tests for display.js because it uses process.stdout
import { dirname } from 'dirname-filename-esm'
import { readFileSync } from 'fs'
import { createHash } from 'crypto'
import { join } from 'path'

const __dirname = dirname(import.meta)

test('okay', () => {
  const hash = createHash('sha256').update(readFileSync(join(__dirname, './display.js'))).digest('hex')
  expect(hash).toMatchSnapshot()
})
