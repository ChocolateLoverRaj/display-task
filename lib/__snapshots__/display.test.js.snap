// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`okay 1`] = `"91390eba15afedd6a6742b054950a62f5487108b7bb0a56ca73c207a4bbe2db1"`;
